styles.css

```diff
@@ -758,3 +758,650 @@ form {
 .darkLine {
   padding: 0 0 8px 135px;
 }
+
+.dskNav {
+  display: none;
+}
+
+.carddsk {
+  display: none;
+}
+
+/* .abtSection hr{
+  visibility: hidden;
+} */
+
+/* .workSecDsk{
+  display: none;
+} */
+
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] While we are working on the desktop version, your mobile version should look fantastic as well  :wink: I just discovered that your mobile version is unsightly since it is not responsive; your project's mobile version pages are not fitting nicely in the complete screen; please solve this. :point_down: 
![mobilever](https://user-images.githubusercontent.com/88894525/201471050-b24f51ed-b7c7-4e94-a874-7b660f0d8b0a.png)

Figma design
![figmamobilever](https://user-images.githubusercontent.com/88894525/201471107-831935af-b751-4b2c-a672-f0bcc7da4491.png)

Reply:
Fixed.